src/main/java/backend/ToDoList.java

```diff
@@ -0,0 +1,143 @@
+                + "I have added the following Deadline to your list: "+ description + "\n"
+                + HORIZONTAL_LINE);
+    }
+
+    /**
+     * Creates a new EventTask and adds it to the toDoList.
+     * Prints message to inform user that task has been added.
+     *
+     * @param description name of the task
+     * @param startTime the time that the event starts
+     * @param startTime the time that the event ends
+     */
+    public void createEventTask(String description, String startTime, String endTime){
+        Task event = new EventTask(description, startTime, endTime);
+        toDoList.add(event);
+        System.out.println(HORIZONTAL_LINE + "\n"
+                + "I have added the following Event to your list: "+ description + "\n"
+                + HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints the index and string representation of every task in the toDoList.
+     * Index starts from 0 in ArrayList, but will be printed out as starting from 1.
+     */
+    public void showList() {
+        System.out.println(HORIZONTAL_LINE + "\n");
+        toDoList.forEach((task) ->
+                System.out.println(toDoList.indexOf(task) + 1 + ". " + task.toString() + "\n"));
+        System.out.println(HORIZONTAL_LINE + "\n");
+    }
+
+    /**
+     * Marks tasks in the list as done.
+     * If the task does not exist, informs user that the task does not exist.
+     *
+     * @param index the index of the task in the <code>ArrayList toDoList</code>
+     */
+    public void markTaskAsDone(int index) {
+        try {
+            Task task = toDoList.get(index);
+            task.markAsDone();
+            System.out.println(HORIZONTAL_LINE + "\n");
+            System.out.println("Well done, I have marked " + task + " as done.");
+            System.out.println(HORIZONTAL_LINE + "\n");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println(HORIZONTAL_LINE + "\n");
+            System.out.println("I do not have this task in my list.");
+            System.out.println(HORIZONTAL_LINE + "\n");
+        }
+    }
+
+    /**
+     * Marks tasks in the list as done.
+     * If the task does not exist, informs user that the task does not exist.
+     *
+     * @param index the index of the task in the <code>ArrayList toDoList</code>
+     */
+    public void unmarkTaskAsDone (int index) {
+        try {
+            Task task = toDoList.get(index);
+            task.unmarkAsDone();
+            System.out.println(HORIZONTAL_LINE + "\n");
+            System.out.println("Understood, I have unmarked " + task + " as done.");
+            System.out.println(HORIZONTAL_LINE + "\n");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println(HORIZONTAL_LINE + "\n");
+            System.out.println("I do not have this task in my list.");
+            System.out.println(HORIZONTAL_LINE + "\n");
+            }
+        }
+
+    /**
+     * Delete a task in the list.
```

Comment:
Should be in imperative
```suggestion
     * Deletes a task in the list.
```